test/expected/tablespace.out

```diff
@@ -88,8 +122,94 @@ INNER JOIN _timescaledb_catalog.chunk ch ON (ch.table_name = c.relname);
  _hyper_2_4_chunk | tablespace2
 (4 rows)
 
+--detach tablespace1 from all tables. Due to lack of permissions,
+--should only detach from 'tspace_1dim' (1 tablespace)
+SELECT detach_tablespace('tablespace1');
+ detach_tablespace 
```

Comment:
I think we should get a warning/notice here.

Reply:
I really don't get why that would be useful. Can you elaborate? We would only warn about things that the user has no association with.

Reply:
Described above. The user expects tablespace1 to be detached from all hypertables here. It isn't.